test/SourceKit/CursorInfo/cursor_some_type.swift

```diff
@@ -35,7 +35,8 @@ func test(value: S) {
 // PROTO_AFTER_SOME-NEXT: Proto
 // PROTO_AFTER_SOME-NEXT: s:4Test5ProtoP
 // PROTO_AFTER_SOME-NEXT: source.lang.swift
-// PROTO_AFTER_SOME-NEXT: Proto.Protocol
+// FIXME: InterfaceTypeRequest returns a meta constraint type for protocols.
```

Comment:
I don’t quite understand that Fixme. What do you mean by it? I this something that broke with this change or something that has been broken before?

Reply:
I guess I phrased it poorly, will fix. It was already this way and, I believe, broken (see [the complement FIXME](https://github.com/apple/swift/pull/65731/files/c1179155713e7504033fdf77b410885b64d5a828#diff-5a9439f8a77b136c97b430596bfc4263de54fbf7c0616656bbfa75ff2d10d145R2409)). The interface type of a protocol was always the protocol metatype aka `P.Protocol` (`(any P).Type` with `any` syntax), which is currently modeled as an `ExistentialType` wrapped in a `MetatypeType`. It seems we forgot to audit the request during the remodeling and still evaluate to a `ProtocolType` directly wrapped in a `MetatypeType`, which is not a sensible type in today’s Swift.